### PR TITLE
Fix several tests and implement several more

### LINKED ISSUE #1405: Integration tests re-enabled by #1235 don't work correctly
Note PR #1404 in which the tests were completely disabled.

In order to resolve this issue, we must not only re-enable the tests, but ensure that either the tests are fixed or the code is fixed.

### PR DESCRIPTION
**Pull request by mhagen-vmware**
Fixes #1405 and #1406

### CODE REVIEW

**Comment by corrieb** [comment]
LGTM

File: tests/test-cases/Group1-Docker-Commands/1-17-Docker-Network-Connect.robot [hunk 1]
```diff
@@ -8,24 +8,21 @@ Suite Teardown  Cleanup VIC Appliance On Test Server
...
-#    Should Contain  ${output}  eth1
+    ${rc}  ${containerID}=  Run And Return Rc And Output  docker ${params} pull busybox
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${containerID}=  Run And Return Rc And Output  docker ${params} create busybox ifconfig
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} network connect test-network ${containerID}
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start ${containerID}
+    Should Be Equal As Integers  ${rc}  0
+    ${status}=  Get State Of Github Issue  366
```
Comment (hmahmood) [inline tests/test-cases/Group1-Docker-Commands/1-17-Docker-Network-Connect.robot]: Why does this issue need to be fixed for this test to work?

Comment (mhagen-vmware) [inline tests/test-cases/Group1-Docker-Commands/1-17-Docker-Network-Connect.robot]: Issue 366 is only preventing the final verification from working, the rest of the test runs fine and at least verifies that we don't return error codes.  But the final verification is needed to verify that we actually added a second network to within the container.

**Comment by mdubya66** [comment]
lgtm

**Comment by mhagen-vmware** [comment]
.

**Pull request merged**